src/index.html

```diff
@@ -1,13 +1,420 @@
+                class="place__link"
+              >
+                Marina Palms
+              </a>
+              / North Miami Beach, FL 33162
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-6">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/3.jpg"
+                      alt="Photo 3"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-7-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/4.jpg"
+                      alt="Photo 4"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a
+                href="https://goo.gl/maps/PD34DCqophWP9cVF7"
+                class="place__link"
+              >
+                Edition Residens
+              </a>
+              / Miami Beach, FL 33139
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/5.jpg"
+                      alt="Photo 5"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-5-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/6.jpg"
+                      alt="Photo 6"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description">
+              <a
+                href="https://goo.gl/maps/2pscDmawHUhLiw569"
+                class="place__link"
+              >
+                Faena House
+              </a>
+              / Miami Beach, FL 33140
+            </div>
+          </div>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--6-7">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section testimonials" id="testimonials">
+        <div class="container">
+          <h2 class="page__section-title">
+            Testimonials
+          </h2>
```

Comment:
Pictures in Testimonials should increase on hover, too
![image](https://user-images.githubusercontent.com/37161315/170643825-5634cf43-0b96-4c83-ade4-d3f6d3b49556.png)